### PR TITLE
 Added code of daily Coding Exercise -9

### PR DESCRIPTION
**Pull request by anushreeSonak**

### CODE REVIEW

File: First/dailyExercises/PrintEvenLength.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        // Splits String
+        for (String word : s.split(" "))
+            // if length is even
+            if (word.length() % 2 == 0)
+                // Print the word
+                System.out.println(word);
+    }
+
+    public static void main(String[] args) {
+        String s = "I am Anushree sonak and I am from kothrud pune";
```
Comment (amitsahu8) [inline First/dailyExercises/PrintEvenLength.java]: Please follow coding guidelines

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

File: First/dailyExercises/PrintEvenLength.java (hunk 1 above)
Comment (anushreeSonak) [reply First/dailyExercises/PrintEvenLength.java]: Done !

File: First/dailyExercises/PrintEvenLength.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package firstPackage.First.dailyExercises;
+
+public class PrintEvenLength {
+    public static void printWords(String s) {
+        // Splits String
+        for (String word : s.split(" "))
```
Comment (amitsahu8) [inline First/dailyExercises/PrintEvenLength.java]: please don't use single character variable name

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

File: First/dailyExercises/PrintEvenLength.java (hunk 2 above)
Comment (anushreeSonak) [reply First/dailyExercises/PrintEvenLength.java]: Done !!

**Review by amitsahu8 — APPROVED** [review]

**Pull request merged**